### PR TITLE
dev:  Dark Mode support for Breadcrumbs

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-1150

## Proposed changes
Adds the usage of the BreadcrumbNav component, and dark mode support for Breadcrumbs

---

## Reviewer notes
Run the app and visit either `/news-announcements`, `/about-us` or `/about-us/orbit-blog` and verify that the styles are accurate. You can also view the `BreadcrumbNav` component in storybook, but the text color is wrong for some reason(working on a solution).

## Setup
Make sure to have both the CMS and the portal client running on your local machine. You can do this by
- Running `yarn services:up && yarn dev` in the CMS repo and
- Running `yarn dev` in the portal client with this branch checked out

To toggle between light and dark mode, open the dev tools in your browser when running the portal, then click Application > Local Storage > Set a key/value pair for theme, dark.

---

### As the original developer, I have

- [ ] Met the acceptance criteria, or will meet them in subsequent PRs or stories
  - ... 
- [ ] Created new stories in Storybook if applicable
  - [ ] Checked that all Storybook accessibility checks are passing
- [ ] Created/modified automated unit tests in Jest
  - [ ] Including jest-axe checks when UI changes
- [ ] Created/modified automated E2E tests in Cypress
  - [ ] Including cypress-axe checks when UI changes
  - [ ] Checked that the E2E test build is not failing
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [ ] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - [ ] Followed guidelines for zero-downtime deploys

### As code reviewer(s), I have

- [ ] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [ ] Checked that all code is adequately covered by tests
  - [ ] Checked that the E2E test build is not failing
- [ ] Made it clear which comments need to be addressed before this work is merged
- [ ] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [ ] Checked in the design translated visually
- [ ] Checked behavior
- [ ] Checked different states (empty, one, some, error)
- [ ] Checked for landmarks, page heading structure, and links
- [ ] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [x] On commercial internet in Edge
  - [x] On commercial internet in Firefox
  - [x] On commercial internet in Chrome
  - [x] On commercial internet in Safari
  - [ ] On NIPR in IE11
  - [ ] On NIPR in Firefox
  - [ ] On NIPR in Chrome
  - [ ] On NIPR in Safari
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR

---

### CODE REVIEW

**Comment by jbecker01** [comment]
Looks good!

File: src/pages/news.tsx [hunk 1]
```diff
@@ -44,19 +40,19 @@ InternalNews.getLayout = (page: React.ReactNode) =>
...
-      </BreadcrumbBar>
+      <BreadcrumbNav
+        navItems={[
+          {
+            path: '/',
+            label: <Breadcrumb>Service portal home</Breadcrumb>,
+          },
+          {
+            path: '/news',
+            label: <Breadcrumb>Internal News</Breadcrumb>,
```
Comment (gidjin) [inline src/pages/news.tsx:51]: Question: Looks like the original navigation in this file was for `About us` should this be `About us` too? Though after looking at the pages in the browser looks like internal news isn't linked yet in the menu so maybe this isn't something we need to worry about right now.

**Review by gidjin — APPROVED** [review]
praise: Thanks for refactoring to use the `BreadcrumbNav` component. I didn't know it existed.

**Review by abbyoung — APPROVED** [review]

File: src/pages/news.tsx (hunk 1 above)
Comment (jcbcapps) [reply src/pages/news.tsx:51]: Ah I missed that. I don't think it would hurt to add that in.

**Pull request merged**